content/en/security_platform/application_security/getting_started/java.md

```diff
@@ -31,13 +31,15 @@ further_reading:
 
    Or one of the following methods, depending on where your application runs:
 
+**Note:** Read-only file systems are not currently supported. The application must have access to a writable `/tmp` directory.
```

Comment:
It need not necessarily be `/tmp`, though that is the usual default. It can be overridden by setting the `java.io.tmpdir` system property).